api-reference/10 UI Widgets/dxDataGrid/1 Configuration/rowTemplate.md

```diff
@@ -13,41 +13,41 @@ Specifies a custom template for rows.
 The [Row](/api-reference/10%20UI%20Widgets/dxDataGrid/6%20Row '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Row/') object extended by the **DataGrid**'s instance (the **component** field) and column configuration (**columns**).
 
 ##### field(rowInfo.columns): Array<dxDataGridColumn>
-<!-- Description goes here -->
+Contains an array of grid columns. Each array item is an object with the column settings. The order of columns is the same as in the [columns](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/columns '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/') array.
 
 ##### field(rowInfo.component): dxDataGrid
-<!-- Description goes here -->
+Contains the **DataGrid** instance.
 
 ##### field(rowInfo.data): any
-<!-- Description goes here -->
+Contains the object of the data source represented by the current row.
 
 ##### field(rowInfo.groupIndex): Number
-<!-- Description goes here -->
+Contains the group index of the current row. Useful if the **rowType** field is *'group'*.
```

Comment:
Try to avoid adjective like "useful" because there may be cases when the thing that you define as useful can turn out to be useless. Do you mean that this field has value only if rowType is "group"?